content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -333,6 +365,36 @@ Support for the `PodResourcesLister service` requires `KubeletPodResources`
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) to be enabled.
 It is enabled by default starting with Kubernetes 1.15 and is v1 since Kubernetes 1.20.
 
+### `Get` gRPC endpoint {#grpc-endpoint-get}
+
+{{< feature-state state="alpa" for_k8s_version="v1.27" >}}
+
+The `Get` endpoint provides information on resources of a running pod. It exposes similar information as
+describe in the `List` endpoint. The `Get` endpoint requires `pod_name` and `pod_namesapce` of the running
+pod.
```

Comment:
By the way, the design to use `_` in the field name is a bad decision against the convention to use camelCase names.

Reply:
so this pod_namespace and pod_name taken from the proto file syntax which is underscore. For golang it will be camelCase according to golang conventions. I updated in the commit.

Reply:
> @moshe010 Would you mind responding to @tengqm when you get the chance? Tomorrow, April 4th, is the **Docs complete — All PRs reviewed and ready to merge** deadline.

sorry for the late response I was OOO on Monday.